opensrp-chw-core/src/main/res/layout/navigation_view.xml

```diff
@@ -0,0 +1,432 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@LZRS Move this class to CHW and make it just specific to LMH flavor

Reply:
I moved them, they are part of this pr https://github.com/opensrp/opensrp-client-chw/pull/1927